scripts/qa_crowbarsetup.sh

```diff
@@ -3404,13 +3404,27 @@ function onadmin_proposal
         done
     fi
     local proposal
-    for proposal in nfs_client pacemaker database rabbitmq keystone swift \
-        ceph monasca glance cinder neutron nova `horizon_barclamp` \
-        ceilometer heat manila trove barbican magnum sahara murano \
-        aodh tempest; do
+    for proposal in nfs_client pacemaker database rabbitmq ; do
         deploy_single_proposal $proposal
     done
 
+    # The database and rabbitmq proposals are known to cause crm
+    # resource failures in HA setups, so apply this temporary cleanup
+    # to avoid failing the whole run.
+    # https://bugzilla.suse.com/show_bug.cgi?id=1030822
+    onadmin_cleanup_db_mq_vips
```

Comment:
It's not reverted yet -  https://github.com/SUSE-Cloud/automation/pull/1882#issuecomment-294697734

Happy to remove it once the revert lands.